proposals/extensions.md

```diff
@@ -995,6 +977,67 @@ extension member lookup will find the `int` property and stop there.
 
 For context see [extension method invocation rules](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#11783-extension-method-invocations).
 
+### Less specific extension type
+
+If `X` extends `C` and `Y` extends `D`, `X` is considered less specific than `Y`
+when:
+- `C` is a base type of `D`, or
+- `C` is an interface implemented by `D`.
+
+### Less specific extension member
+
+TL;DR: As part of extension member lookup, extension invocations and overload resolution,
+we consider that "less specific" extension members are "hidden" by "more specific" extension members.
```

Comment:
It looks like the rules are specified in terms of specificity of extension type instead. It also looks like the rule specified in the paragraph below is not used anywhere. Perhaps that paragraph should be removed, and the rest of the current section (the examples, etc.) should be merged into the previous section